src/core/renderer.js

```diff
@@ -200,23 +200,6 @@ geo.renderer = function (arg) {
   this._render = function () {
   };
 
-  ////////////////////////////////////////////////////////////////////////////
-  /**
-   * Exit
-   */
-  ////////////////////////////////////////////////////////////////////////////
-  this._exit = function () {
```

Comment:
why not we keep the _exit API on the renderer and use it for the purpose of cleaning up and call it from the deleteCache? Also, having this API is consistent with the rest of the objects.